relayer/processor/path_processor.go

```diff
@@ -750,10 +685,10 @@ func (pp *PathProcessor) processLatestMessages(ctx context.Context) error {
 	// now send messages in parallel
 	var eg errgroup.Group
```

Comment:
Should this use `errgroup.WithContext`? I'm not sure if we want to cancel sending messages to the other end if one fails. Either way, a clarifying comment would be helpful here.

Reply:
added a comment, don't need to hold up sending to one end if there's a temporary issue with the other.